test/module/SmartContractOwnershipRegistry.Module.specs.ts

```diff
@@ -0,0 +1,491 @@
+    const EIP1271_MAGIC_VALUE = "0x1626ba7e";
+
+    const setupTests = deployments.createFixture( async( {deployments} ) =>{
+        await deployments.fixture();
+
+        const entryPoint = await getEntryPoint();
+        const saFactory = await getSmartAccountFactory();
+        const smartContractOwnershipRegistryModule = await getSmartContractOwnershipRegistryModule();
+        const ecdsaRegistryModule = await getEcdsaOwnershipRegistryModule();
+        const mockToken = await getMockToken();
+
+        let ecdsaOwnershipSetupData1 = ecdsaRegistryModule.interface.encodeFunctionData(
+            "initForSmartAccount",
+            [baseSmartAccountOwner1.address]
+        );
+        let ecdsaOwnershipSetupData2 = ecdsaRegistryModule.interface.encodeFunctionData(
+            "initForSmartAccount",
+            [baseSmartAccountOwner2.address]
+        );
+        const smartAccountOwnerContract1= await getSmartAccountWithModule(ecdsaRegistryModule.address, ecdsaOwnershipSetupData1, smartAccountDeploymentIndex);
+        const smartAccountOwnerContract2= await getSmartAccountWithModule(ecdsaRegistryModule.address, ecdsaOwnershipSetupData2, smartAccountDeploymentIndex+1);
+
+        let smartContractOwnershipSetupData = smartContractOwnershipRegistryModule.interface.encodeFunctionData(
+            "initForSmartAccount",
+            [smartAccountOwnerContract1.address]
+        );
+        const userSA = await getSmartAccountWithModule(smartContractOwnershipRegistryModule.address, smartContractOwnershipSetupData,smartAccountDeploymentIndex+2);
+        await deployer.sendTransaction({
+            to: userSA.address,
+            value: ethers.utils.parseEther("60"),
+        });
+        const tokensToMint = ethers.utils.parseEther("100");
+        await mockToken.mint(userSA.address,tokensToMint.toString());
+        await mockToken.mint(bob.address,tokensToMint.toString());
+
+        const randomContractCode = `
+            contract random {
+                function returnAddress() public view returns(address){
+                    return address(this);
+                }
+            }
+            `;
+        const randomContract = await deployContract(deployer,randomContractCode);
+
+        return {
+            entryPoint: entryPoint,
+            saFactory: saFactory,
+            smartContractOwnershipRegistryModule: smartContractOwnershipRegistryModule,
+            ecdsaRegistryModule: ecdsaRegistryModule ,
+            mockToken: mockToken,
+            userSA: userSA,
+            smartAccountOwnerContract1: smartAccountOwnerContract1,
+            smartAccountOwnerContract2: smartAccountOwnerContract2,
+            smartContractOwnershipSetupData: smartContractOwnershipSetupData,
+            randomContract: randomContract
+        }
+    });
+
+    describe("initForSmartAccount(): ", async()=>{
+        it("Calls initForSmartAccount and it successfully initializes userSA with Smart Contract as Owner", async()=>{
```

Comment:
initForSmartAccount(): is already in `describe`, no need to put it in `it`
`describe` contains info about which function and `it` contains info about expected behaviour
so 
initForSmartAccount 
     should successfully initialize userSA with Smart Contract as Owner